package.json

```diff
@@ -0,0 +1,38 @@
+{
+	"name": "inferno-backend",
+	"version": "1.0.0",
+	"description": "A token distributor for the Inferno Rewards program",
+	"main": "src/index.ts",
+	"repository": "git@github.com:ardriveapp/inferno-backend.git",
+	"author": {
+		"name": "Permanent Data Solutions Inc",
+		"email": "info@ardrive.io",
+		"website": "https://ardrive.io"
+	},
+	"license": "AGPL-3.0",
+	"scripts": {
+		"clean": "rimraf [ lib .nyc_output node_modules coverage ]",
+		"format": "prettier --write \"src/**/*.ts\"",
+		"lint": "eslint . --ext .ts",
+		"lintfix": "eslint . --ext .ts --fix",
+		"typecheck": "tsc --noemit",
+		"build": "yarn clean && tsc --project ./tsconfig.prod.json",
+		"dev": "yarn clean && tsc --project ./tsconfig.prod.json -w"
+	},
+	"devDependencies": {
+		"@types/eslint": "^8",
+		"@types/rimraf": "^3",
```

Comment:
why we need rimraf types?

Reply:
ccc72f5bbdaaa17c26390700ad7c50a35ff2d6f8